### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by ThakareSakshi**
added swift tutorial in Hindi by code cat

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Added swift programming course in Hindi

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -618,6 +618,7 @@
 
 ### Swift
 
+* [Swift programming hindi tutorial for beginners](https://www.youtube.com/playlist?list=PLb5R4QC2DtFv3MvfG42Cd5La34Hwj4pY6) - Code Cat
```
Comment (3R1Dev) [inline courses/free-courses-hi.md:621]: Playlist check out, I would suggest changing the title to be Uppercase as in the suggestion below.

```suggestion
* [Swift Programming Hindi Tutorial for Beginners](https://www.youtube.com/playlist?list=PLb5R4QC2DtFv3MvfG42Cd5La34Hwj4pY6) - Code Cat
```

**Review by 3R1Dev — CHANGES_REQUESTED** [review]
Update the title of the course.

**Review by 3R1Dev — APPROVED** [review]
Looks good, thanks for the contribution!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**